src/osx/cocoa/utils.mm

```diff
@@ -72,6 +72,9 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
     wxUnusedVar(notification);
     [NSApp stop:nil];
     wxTheApp->OSXOnDidFinishLaunching();
+
+    [NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
+    [NSApp activateIgnoringOtherApps: YES];
```

Comment:
Does this still work if you do it inside `if ( !app.EndsWith(".app") )` as was done in the old code being removed by this PR? I think it should and it seems it would be better to avoid messing with activation when the app is bundled.

Reply:
absolutely, we must only trigger this for the non-standard cases. I'll look at it

Reply:
+1. I was testing with [this](https://gist.github.com/discnl/81223e18bbce96440facdd48f5248012) (about the same but applying before `NSApp stop:nil`). It also still works with 10.12 and am going to try to test on 10.10 with builds from a more recent setup.

Reply:
I couldn't easily test with builds from newer dev tools so I tested on the minimal supported 10.10 with builds from Xcode 10. Still seems to behave the same as before when running as non-bundled app so there's a working menu and dock icon. I'm also not seeing any difference regarding `MacOpenFiles` (via mediactrl sample) and DnD also still works.
Tested on 10.10, 10.12, 10.15 and 11.0.

Reply:
thanks, then I'm fine with that, I'll regroup the code afterwards to make sure we only have it once, using something along a "OSXIsAppBundled" method, if that's ok @vadz

Reply:
> thanks, then I'm fine with that, I'll regroup the code afterwards to make sure we only have it once, using something along a "OSXIsAppBundled" method, if that's ok @vadz

Please note that my testing was done with [this](https://gist.github.com/discnl/81223e18bbce96440facdd48f5248012) mentioned diff which does things before `[NSApp stop:nil];` / `OSXOnDidFinishLaunching()` and not after. I don't know if either order is preferred.

if you're only adding a `OSXIsAppBundled()` to make sure we have the code once as you say: it already only appears once in wx, it just moved this time. Ignore if it's intended for users also, though I don't know of a demand for it.

Reply:
Thanks. Sorry, I didn't realise the entire bringToFront part was not needed anymore, if the current implementation does so under all the systems, then perfect, and no need for a common method :-)